src/kubernetes-tool/utils/helm_parts/document_parser.ts

```diff
@@ -84,12 +115,17 @@ export default class DocumentParser {
         for (const q of additionalQuotes) args.push(q)
 
         // Gets the function.
-        const func: string = args.shift()! as string
+        let func = args.shift()! as string
+        if (((func as unknown) as Quote).text) {
+            func = ((func as unknown) as Quote).text
+        }
 
         // Runs the function.
         if (functions[func] === undefined) {
             if (func.startsWith(".")) return String(this.helmdef2object(func))
-            throw new Error(`${func} - Unknown command!`)
+
+            // We should return here because even though this may not be fully accurate, it allows for an as accurate as possible result.
+            return ""
```

Comment:
What does this look like to the user?

Reply:
Won't show that one line. This is fine since a lot of the core functionality is implemented, and it seems like a better idea in the event that a function is missing to just ignore it.